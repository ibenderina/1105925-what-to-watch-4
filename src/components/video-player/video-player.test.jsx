import VideoPlayer from "@components/video-player/video-player";

it(`Should VideoPlayer render correctly`, () => {
  const videoUrl = `Hello`;
  const filmImage = `Hello`;

  let tree;
  window.act(() => {
    tree = window.create(<VideoPlayer
      videoUrl={videoUrl}
      filmImage={filmImage}
      isMuted={true}
      isPlayed={true}
    />);
  });
  expect(tree.toJSON()).toMatchSnapshot();
});
