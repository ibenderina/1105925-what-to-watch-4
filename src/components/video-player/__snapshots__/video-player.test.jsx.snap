// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should VideoPlayer render correctly 1`] = `
<video
  autoPlay={true}
  height="175"
  loop={true}
  muted={true}
  poster="Hello"
  src="Hello"
  width="280"
/>
`;
